config/default.json

```diff
@@ -2,7 +2,7 @@
   "mode": "bundled",
   "port": 8081,
   "anchorControllerEnabled": false,
-  "cronExpression": "0/5 * * * ? *",
+  "schedulerIntervalMS": 300000,
```

Comment:
this isn't the interval when batches run, but just the interval for when the scheduled checks if a batch *should* run, right?  In that case should we make the interval shorter?  Like every minute?  Or even every 10 seconds?

Reply:
Yes I agree. I will be changing this once we are using the scheduler service to emit anchor events. Currently this interval is set to the same amount of time as the cronExpression to preserve current functionality